src/test/java/bio/terra/controller/DatasetOperationTest.java

```diff
@@ -184,9 +218,77 @@ public void deleteAllBigQueryProjects() throws Exception {
         }
     }
 
-    private DatasetSummaryModel createTestDataset(DatasetRequestModel datasetRequest, String infix) throws Exception {
-        String baseName = datasetRequest.getName();
-        String datasetName = randomizedName(baseName, infix);
+    private StudySummaryModel setupMinimalStudy() throws Exception {
+        StudySummaryModel studySummary = createTestStudy("study-minimal.json");
+        loadStudyData(studySummary.getName(), "participant", "study-minimal-participant.csv");
+        loadStudyData(studySummary.getName(), "sample", "study-minimal-sample.csv");
+        return  studySummary;
+    }
+
+    // create a study to create datasets in and return its id
+    private StudySummaryModel createTestStudy(String resourcePath) throws Exception {
+        String studyJson = IOUtils.toString(classLoader.getResourceAsStream(resourcePath));
+        StudyRequestModel studyRequest = objectMapper.readerFor(StudyRequestModel.class).readValue(studyJson);
+        studyRequest.setName(randomizedName(studyRequest.getName(), ""));
+
+        MvcResult result = mvc.perform(post("/api/repository/v1/studies")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(objectMapper.writeValueAsString(studyRequest)))
+                .andExpect(status().isCreated())
+                .andReturn();
+
+        MockHttpServletResponse response = result.getResponse();
+        StudySummaryModel studySummaryModel =
+                objectMapper.readValue(response.getContentAsString(), StudySummaryModel.class);
+
+        return studySummaryModel;
+    }
+
+    private void loadStudyData(String studyName, String tableName, String resourcePath) throws Exception {
+        String datasetName = PDAO_PREFIX + studyName;
+        String location = "US"; // ?? Hope this will work. It is what
```

Comment:
This comment is poetic 😄 I was wondering the same thing when I was testing out the bq cli for ingesting.